### ISSUE TITLE
After Reboot of the Pi, KlipperScreen isn't showing up

### ISSUE THREAD

**Issue by nicedevil007** [th33xitus/kiauh#61]
I reinstalled KlipperScreen with the "pi" User with your script after I deleted the systemd service that was installed with the root user (don't know if that was neccessary).

After a reboot, the screen now only shows the shell login screen instead of KlipperScreen

**Comment by jordanruthe** [comment]
What base os did you use? Raspbian lite, raspbian, or something else?

**Comment by nicedevil007** [comment]
> What base os did you use? Raspbian lite, raspbian, or something else?

I'm using Raspbian OS with Desktop Env, but that one was uninstalled because I found Klipper Screen (no need for Fluidd Kioskmode anymore :) ).

After uninstallation of the desktop env, I installed KlipperScreen. When I do this with the "pi" user the thing from above happens. Yesterday I retryed installing it with "root" and it works like a charm.

Just want to have everything in KIAUH to be have one page to update everything that is needed to drive a printer with Klipper :)

When I run your script from your repository from this location `~/KlipperScreen/scripts/KlipperScreenInstall.sh` where did it install the folder for the software itself? Maybe I deleted the wrong folder last time and that can cause the issue that the "pi" user can't overwrite files that belong to "root"?
